### ISSUE TITLE
VREP can't handle object

### ISSUE THREAD

**Issue by ThanasisTs** [BenjaminNavarro/open-phri#3]
Hello,

I am building the OpenPHRI library including the examples by making a change in the `standalone_install.sh` file (changed the value of the variable `with_examples` to ON). When I run an example (path_follower_example or the demo), the program crushes logging to the console that it can't get the handle of the object  LBR4p_tcp. If I understand correctly, this problem arises because the `simxGetObjectHandle` function can't locate the LBR4p_tcp object. Is there a way to fix this issue? Thanks in advance and please let me know if there is any additional information you would like to provide you.

**Comment by BenjaminNavarro** [comment]
Are you using one of the V-REP scenes provided with OpenPHRI? (inside [share/scenes](https://github.com/open-phri/open-phri/tree/master/share/scenes))

I just realized another issue though is that the robot disappears when the simulation is started using Newton as physics engine. It's probably a V-REP bug since it occurs even importing a LWR4 model inside a new scene.

**Comment by ThanasisTs** [comment]
Yes you are right, I wasn't loading the scene in the V-REP scene. Thank you. However I noticed that even when loading the scene and running the demo application, the program still fails logging the same output. This does not happen with the rest applications. Lastly I would like to ask if there is any kind of documentation of OpenPHRI so it would be easier to fully understand its capabilities. Thanks in advance.

**Comment by ThanasisTs** [comment]
Yes you are right, i wasn't loading the V-REP scene. However, I noticed that even when I am loading the scene, the demo application still fails logging the same error. The rest examples work fine.
